deb/openmediavault/workbench/src/app/pages/services/nfs/nfs-share-form-page.component.ts

```diff
@@ -69,7 +69,9 @@ export class NfsShareFormPageComponent extends BaseFormPageComponent {
           gettext(
             // eslint-disable-next-line max-len
             "Please check the <a href='https://manpages.debian.org/nfs-kernel-server/exports.5.html' target='_blank'>manual page</a> for more details."
-          ),
+          ) +
+          ' ' +
+          gettext("WARNING: UI only supports a single client entry per share."),
```

Comment:
This is incorrect. You can limit the number of clients/hosts via prefix length.

1 host: 192.0.2.0/30
~100 hosts: 192.0.2.0/25
255 host: 192.0.2.0/24

Reply:
I tried to get that point across by stating "single client entry" instead of "single client" but I guess that's too easy to overlook/misunderstand. What about "UI does not support multiple comma-separated values", then?

Reply:
> What about "UI does not support multiple comma-separated values", then?

This issue is not only related to the UI. This is how the configuration pipeline of OMV works for NFS.

Reply:
And this is not a `WARNING`, it's only a `NOTE`.

Reply:
> Perhaps `Note: Only one host can be configured per share` is more appropriate.

You said yourself that 192.0.2.0/25 is ~100 hosts so this does not seem to clear up the confusion at all?

> > What about "UI does not support multiple comma-separated values", then?
> 
> This issue is not only related to the UI. This is how the configuration pipeline of OMV works for NFS.

Let's focus on OMV, then, by changing "UI" to "OMV". From the point of view of the user, the NFS manual (linked from the field text) says whitespace-separated values are supported when in fact they are not supported by OMV. This is the source of confusion.

Reply:
`host` is the name used in the man page for describing how the /etc/exports line is constructed.

> Let's focus on OMV, then, by changing "UI" to "OMV".

This is totally superfluous because the user is in the OMV UI. Therefore it is not necessary to mention this again.

Reply:
It's true that the user is in the OMV UI but at the same time the user is also reading a hint text that mentions the NFS manual. It needs to be clear that there is discrepancy between what NFS supports and what OMV supports (via the Client field) so saying "OMV UI" in this case is not superfluous.